modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/AndesConfigurationManager.java

```diff
@@ -168,6 +169,11 @@ public static void initialize(int portOffset) throws AndesException {
             // set carbonPortOffset coming from carbon
             AndesConfigurationManager.carbonPortOffset = portOffset;
 
+            //set delivery timeout for a message
+            int deliveryTimeoutForMessage = AndesConfigurationManager.readValue(AndesConfiguration
+                    .PERFORMANCE_TUNING_TOPIC_MESSAGE_DELIVERY_TIMEOUT);
+            AndesContext.getInstance().setDeliveryTimeoutForMessage(deliveryTimeoutForMessage);
```

Comment:
Can we read and store this property in AndesContext constructor instead of using a setter ? 

assigning it here breaks the intended separation from configs / functional classes.